src/kinetics/KineticsFactory.cpp

```diff
@@ -168,17 +171,61 @@ void addReactions(Kinetics& kin, const AnyMap& phaseNode, const AnyMap& rootNode
             AnyMap reactions = AnyMap::fromYamlFile(fileName,
                 rootNode.getString("__file__", ""));
             for (const auto& R : reactions[node].asVector<AnyMap>()) {
-                kin.addReaction(newReaction(R, kin));
+                try{
+                    kin.addReaction(newReaction(R, kin));
+                }
+                catch (CanteraError& err){
```

Comment:
These formatting suggestions apply here and elsewhere:
```suggestion
                try {
                    kin.addReaction(newReaction(R, kin));
                } catch (CanteraError& err) {
```